fpga/mqnic/ZCU106/fpga_zynqmp/ps/petalinux/project-spec/configs/config

```diff
@@ -290,7 +290,7 @@ CONFIG_YOCTO_NETWORK_SSTATE_FEEDS=y
 #
 # Network sstate feeds URL
 #
-CONFIG_YOCTO_NETWORK_SSTATE_FEEDS_URL="http://petalinux.xilinx.com/sswreleases/rel-v${PETALINUX_VER%%.*}/aarch64/sstate-cache"
+CONFIG_YOCTO_NETWORK_SSTATE_FEEDS_URL="http://petalinux.xilinx.com/sswreleases/rel-v2021.1/aarch64/sstate-cache"
```

Comment:
Did some script make this change?  Seems to me that keeping things as version independent as possible is the way to go.  This change is going to be a problem if it prevents building on 2021.2, for example.

Reply:
I wasn't sure which version of Vivado/petalinux should be used when starting this project. The documentation Getting Started calls out two different versions, vivado = 2020.2, petalinux = 2021.1. Since most of these  things are ironed out for 2021.2, maybe it is best to update the Documentation Getting Started to be specific on which version of Vivado/Petalinux to use when building for the ZCU106. Just a thought.

Reply:
Hmmm, I've never touch this configuration item, `CONFIG_YOCTO_NETWORK_SSTATE_FEEDS_URL` in the past. Always had it like PetaLinux itself set it on project creation. But now that I'm looking at this shell variable expansion: `${PETALINUX_VER%%.*}`, it seems like the generated default is not that generic, since the expansion turns `2021.2` for example, into `2021`. When looking at the [Xilinx server](http://petalinux.xilinx.com/sswreleases/), one can see, that they have subdirectories for any version without `.<something>` at the end _and_ they have subdirectories with the complete version `<something>.<something>`. Thus I'm wondering whether in fact the `%%.*` part of the expansion should be removed - to have it point _the_ specific version subdirectory in question.

And the question for Xilinx is: What's the difference between subdirectories `2021` and `2021.1` or `2021.2` on their server? Is `2021` always the latest? Which would not make sense, when someone is using `2021.1` explicitly and has the above mentioned expansion.

Regardless of the finding above, I agree with Alex, that it makes sense to keep that URL "generic" and not have any hard-coded version number here, unless there is a very specific, clear and understood reason.